.travis/test_script.sh

```diff
@@ -0,0 +1,15 @@
+PR_DIR=$PWD
+
+pdk new module pr_new_modules --template-url=$PR_DIR --skip-interview
+
+pdk new module normal_module --skip-interview
+cd normal_module
+pdk convert --template-url=$PR_DIR --skip-interview --force
+cat convert_report.txt
```

Comment:
Not sure I need to output this convert_report.  The one below for difference from HEAD is more useful.